### PR TITLE
Service Area Task - update UI

### PR DESCRIPTION
**Pull request by JonLavi**
I think we could lay out the UI of this sample a bit neater. Currently the buttons are all in a long row without any space. 

This PR puts the buttons in a control area, and re-arranges the buttons slightly so that it's clearer that the toggle group buttons (create facility, create barrier) are related. 

It's a bit of extra FXML code but I do think it makes the sample easier to use and it looks much smoother

### CODE REVIEW

File: network_analysis/service-area-task/src/main/resources/service_area_task.fxml [hunk 1]
```diff
@@ -15,31 +15,55 @@
...
+        </background>
+
+        <HBox>
+            <ToggleButton fx:id="btnAddFacility" text="Add Facilities" minWidth="90">
+                <toggleGroup>
+                    <ToggleGroup fx:id="toggleGroup"/>
+                </toggleGroup>
+            </ToggleButton>
+            <ToggleButton fx:id="btnAddBarrier" toggleGroup="$toggleGroup" text="Add Barrier" onAction="#createBarrier"
+                          minWidth="90"/>
```
Comment (tschie) [inline network_analysis/service-area-task/src/main/resources/service_area_task.fxml]:
```suggestion
            <fx:define>
                <ToggleGroup fx:id="toggleGroup"/>
            </fx:define>
            <ToggleButton fx:id="btnAddFacility" text="Add Facilities" minWidth="90" toggleGroup="$toggleGroup"/>
            <ToggleButton fx:id="btnAddBarrier" text="Add Barrier"  minWidth="90"  toggleGroup="$toggleGroup" onAction="#createBarrier"/>
```

Comment (tschie) [reply network_analysis/service-area-task/src/main/resources/service_area_task.fxml]: I think using define is a bit cleaner

File: network_analysis/service-area-task/src/main/resources/service_area_task.fxml [hunk 2]
```diff
@@ -15,31 +15,55 @@
...
+        </HBox>
+        <Button text="Show Service Areas" onAction="#showServiceAreas" minWidth="180"/>
+        <Button text="Reset" onAction="#clearRouteAndGraphics" minWidth="180"/>
```
Comment (tschie) [inline network_analysis/service-area-task/src/main/resources/service_area_task.fxml]: Is there a way we can clean up all of these min/max widths? Not a big deal if not.

**Review by tschie — APPROVED** [review]
This is a welcome change. Just one suggestion.

File: network_analysis/service-area-task/src/main/resources/service_area_task.fxml (hunk 1 above)
Comment (JonLavi) [reply network_analysis/service-area-task/src/main/resources/service_area_task.fxml]: Definitely, thanks for pointing that out!

File: network_analysis/service-area-task/src/main/resources/service_area_task.fxml (hunk 2 above)
Comment (JonLavi) [reply network_analysis/service-area-task/src/main/resources/service_area_task.fxml]: Good point, I've gone ahead and put all the styling in the CSS file. Perhaps we can do this with samples going forward? Makes the FXML look nice and clean.

**Review by Rachael-E — APPROVED** [review]
cool, looks good!

**Comment by JonLavi** [comment]
Thanks both!

**Pull request merged**